pages.fr/common/rsync.md

```diff
@@ -4,34 +4,30 @@
 > Peut transférer un ou plusieurs fichiers correspondant à un motif.
 > Plus d'informations : <https://download.samba.org/pub/rsync/rsync.1>.
 
-- Transférer un fichier local vers un serveur distant :
+- Transférer un fichier :
 
-`rsync {{chemin/vers/fichier_local}} {{hote_distant}}:{{chemin/vers/dossier_distant}}`
+`rsync {{chemin/vers/origine}} {{chemin/vers/destination}}`
 
-- Transférer un fichier d'un serveur distant vers l'hôte local :
+- Utiliser le mode archive (copier récursivement les répertoires, copier les liens symboliques sans résolution et conserver les autorisations, la propriété et les délais de modification) :
 
-`rsync {{hote_distant}}:{{chemin/vers/fichier_distant}} {{chemin/vers/dossier_local}}`
+`rsync --archive {{chemin/vers/origine}} {{chemin/vers/destination}}`
 
-- Transférer un fichier sous forme d'[a]rchive (pour conserver les attributs) et compressé ([z]ippé), en mode [v]erbeux, lisible par l'[h]umain et afficher la [p]rogression du transfert :
+- Transférer le contenu d'un dossier :
 
-`rsync -azvhP {{chemin/vers/fichier_local}} {{hote_distant}}:{{chemin/vers/dossier_distant}}`
+`rsync --recursive {{chemin/vers/origine}} {{chemin/vers/destination}}`
 
-- Transférer un dossier et tous ses sous-dossiers d'un hôte distant vers l'hôte local :
+- Transférer le contenu d'un dossier (mais pas le dossier lui-même) :
 
-`rsync -r {{hote_distant}}:{{chemin/vers/dossier_distant}} {{chemin/vers/dossier_local}}`
+`rsync --recursive {{chemin/vers/origine}}/ {{chemin/vers/destination}}`
 
-- Transférer le contenu d'un dossier (mais pas le dossier lui-même) d'un hôte distant vers un hôte local :
+- Utiliser le mode archive, résolvant les liens symboliques et ignorant les fichiers déjà transférés sauf si plus récents :
 
-`rsync -r {{hote_distant}}:{{chemin/vers/dossier_distant}}/ {{chemin/vers/dossier_local}}`
+`rsync --archive --update --copy-links {{chemin/vers/origine}} {{chemin/vers/destination}}`
 
-- Transférer un dossier [r]écursivement, dans une [a]rchive pour conserver les attributs, en résolvant les [l]iens symboliques, et ignorant les fichiers déjà transférés sa[u]f si plus récents :
+- Transférer un fichier vers un hôte distant exécutant `rsyncd` et supprimez les fichiers sur la destination qui n'existent pas sur l'hôte distant :
```

Comment:
I don't understand, it seems incorrect. :thinking:

Reply:
Suggestion (but not sure if correct):
> - Transférer un fichier vers un hôte distant via `rsyncd` en supprimant les fichiers distants qui n'existent pas dans la source :

Reply:
I agree with @nicokosi. It sounds better